### PR TITLE
feat: #755 Add styles for home page

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/src/component/button/Button.module.scss [hunk 1]
```diff
@@ -123,3 +123,20 @@
 .disabled {
   opacity: $disableOpacity;
 }
+
+.homeButton {
```
Comment (tritonJS826) [inline mw-webapp/src/component/button/Button.module.scss:127]: superSpecialBeautifulButton

File: mw-webapp/src/logic/homePage/HomePage.tsx [hunk 2]
```diff
@@ -14,38 +18,153 @@ export const HomePage = () => {
...
-        <GoalItem
-          title={LanguageService.home.goals.goalOne.title[language]}
-          description={LanguageService.home.goals.goalOne.description[language]}
+    <>
+      <VerticalContainer className={styles.welcomeBlock}>
+        {/* <Title
+          level={HeadingLevel.h2}
+          text={LanguageService.home.welcome[language]}
+          className={styles.welcomeTitle}
+        /> */}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/homePage/HomePage.tsx:27]: !

File: mw-webapp/src/logic/homePage/HomePage.tsx [hunk 3]
```diff
@@ -14,38 +18,153 @@ export const HomePage = () => {
...
+        <Title
+          level={HeadingLevel.h3}
+          text={LanguageService.home.description[language]}
+          className={styles.titleDescription}
         />
-        <GoalItem
-          title={LanguageService.home.goals.goalThree.title[language]}
-          description={LanguageService.home.goals.goalThree.description[language]}
+        <Button
+          onClick={() => {}}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/homePage/HomePage.tsx:39]: !

File: mw-webapp/src/logic/homePage/HomePage.tsx [hunk 4]
```diff
@@ -14,38 +18,153 @@ export const HomePage = () => {
...
+              <li>
+                {LanguageService.home.aboutAppBlock.reasonThree[language]}
+              </li>
+            </ul>
+            <div className={styles.aboutProjectDescription}>
+              {LanguageService.home.aboutAppBlock.endDescription[language]}
+            </div>
+          </VerticalContainer>
+          <Button
+            onClick={() => {}}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/homePage/HomePage.tsx:105]: !

File: mw-webapp/src/logic/homePage/HomePage.tsx [hunk 5]
```diff
@@ -14,38 +18,153 @@ export const HomePage = () => {
...
+              <Icon
+                name="WayIcon"
+                size={IconSize.MEDIUM}
+                className={styles.icon}
+              />
+              {LanguageService.home.aboutAppBlock.reasonFour[language]}
+            </HorizontalContainer>
+          </VerticalContainer>
+          <Button
+            onClick={() => {}}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/homePage/HomePage.tsx:155]: !

File: mw-webapp/src/utils/ThemeWorker.ts [hunk 6]
```diff
@@ -137,6 +141,12 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     // [Theme.DARK]: "rgb(108,107,116)",
     // [Theme.LIGHT]: "rgb(200, 194, 194)",
   },
+  homeActiveButtonColor: {
+    [Theme.DARK]: "#F2EFF9",
+    [Theme.LIGHT]: "#F2EFF9",
+    // [Theme.DARK]: "rgb(108,107,116)",
```
Comment (tritonJS826) [inline mw-webapp/src/utils/ThemeWorker.ts:147]: !

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/button/Button.module.scss (hunk 1 above)
Comment (Ekaterina1994) [reply mw-webapp/src/component/button/Button.module.scss]: done

File: mw-webapp/src/logic/homePage/HomePage.tsx (hunk 2 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.tsx]: deleted

File: mw-webapp/src/logic/homePage/HomePage.tsx (hunk 3 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.tsx]: done

File: mw-webapp/src/logic/homePage/HomePage.tsx (hunk 4 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.tsx]: done

File: mw-webapp/src/logic/homePage/HomePage.tsx (hunk 5 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.tsx]: done

File: mw-webapp/src/utils/ThemeWorker.ts (hunk 6 above)
Comment (Ekaterina1994) [reply mw-webapp/src/utils/ThemeWorker.ts]: done

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**